policyengine_api/routes/metadata_routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint
+
+from policyengine_api.utils.payload_validators import validate_country
+from policyengine_api.services.metadata_service import MetadataService
+
+metadata_bp = Blueprint("metadata", __name__)
+metadata_service = MetadataService()
+
+
+@metadata_bp.route("/<country_id>/metadata", methods=["GET"])
```

Comment:
Following the pattern elsewhere, I'd recommend, writing as below. That said, I realize you have some questions about if this entity breakdown is correct; happy to discuss and adjust our process moving forward.
```suggestion
@metadata_bp.route("", methods=["GET"])
```

Reply:
See comment above. Resolving.